### PR TITLE
ASPxSpreadsheet tests're fixed in Chrome and IE

### PR DESCRIPTION
**Pull request by helen-dikareva**
/cc @AlexanderMoskovkin @AndreyBelym 
need some change in testcafe-hammerhead

### CODE REVIEW

**Comment by AlexanderMoskovkin** [comment]
can we add a functional test for this here?

**Comment by helen-dikareva** [comment]
ok

**Comment by helen-dikareva** [comment]
For IE we need to wait for https://github.com/DevExpress/testcafe-hammerhead/pull/821

File: test/functional/fixtures/api/es-next/press-key/test.js [hunk 1]
```diff
@@ -7,10 +7,17 @@ describe('[API] t.pressKey', function () {
...
+            shouldFail: true,
+            only:       'chrome'
+        })
             .catch(function (errs) {
                 expect(errs[0]).to.contain('The "keys" argument is expected to be a non-empty string, but it was boolean.');
                 expect(errs[0]).to.contain('> 19 |    await t.pressKey(false);');
             });
     });
+
+    it('Should press key in iframe', function () {
```
Comment (AlexanderMoskovkin) [inline test/functional/fixtures/api/es-next/press-key/test.js]: I think we should have more informative mocha-test test name.  For example:
```js
it('Should raise event in different windows if focus was changed during action execution ', function () {
    return runTests('./testcafe-fixtures/press-key-test.js', 'Press key in iframe', { only: 'chrome' });
});
```

Comment (helen-dikareva) [inline test/functional/fixtures/api/es-next/press-key/test.js]: agree

**Comment by helen-dikareva** [comment]
@testcafe-build-bot /retest

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
Unstable test which will be fixed in https://github.com/DevExpress/testcafe/pull/842

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
FPR @AndreyBelym

**Comment by helen-dikareva** [comment]
@VasilyStrelyaev please check test name

**Pull request merged**